### PR TITLE
Fix ember-try deprecations

### PR DESCRIPTION
**Pull request by xomaczar**

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -1,55 +1,3 @@
 /*jshint node:true*/
 module.exports = {
-  scenarios: [
```
Comment (DanChadwick) [inline config/ember-try.js]: We need these scenarios for travis-ci, right?

File: package.json [hunk 2]
```diff
@@ -71,6 +71,9 @@
   "ember-addon": {
     "configPath": "tests/dummy/config",
     "demoURL": "https://miguelcobain.github.io/ember-paper",
-    "before": "ember-cli-sri"
+    "before": "ember-cli-sri",
+    "versionCompatibility": {
+      "ember": ">=2.3.0"
```
Comment (DanChadwick) [inline package.json]: Could you please point me to the documentation for this option within `ember-addon`. Google failed me. Or maybe I failed google. ;)

Comment (xomaczar) [inline package.json]: https://github.com/ember-cli/ember-try/blob/master/README.md Config section

File: config/ember-try.js (hunk 1 above)
Comment (xomaczar) [inline config/ember-try.js]: Scenarios are for ember-try, do we not have .travis.yml?

**Comment by DanChadwick** [comment]
@xomaczar Thanks! Funny thing is I was on that page and somehow didn't see it. If you restore the scenarios, we can merge this. Thanks for your contribution!

**Comment by DanChadwick** [comment]
@xomaczar - travis.yml defines which scenarios to run. The scenarios themselves are defined in ember-try.js. At least that's the way it was worked, but I'm no expert. The reason the PR's tests failed is due to the missing scenarios.

**Comment by xomaczar** [comment]
@DanChadwick We don't need to explicitly define scenarios for ember-try, it will use versionCompatibility and generate scenarios based on semver. I fixed the travis, now everything passes.

**Pull request merged**